substrate/frame/proxy/src/weights.rs

```diff
@@ -46,8 +46,7 @@
 #![allow(unused_imports)]
 #![allow(missing_docs)]
 
-use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
-use core::marker::PhantomData;
+use frame::weights_prelude::*;
```

Comment:
this change doesn't make sense. it needs to be done at the template level, or handled later.